test/Core.Test/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/ResetPasswordPolicyRequirementFactoryTests.cs

```diff
@@ -12,23 +13,26 @@ namespace Bit.Core.Test.AdminConsole.OrganizationFeatures.Policies.PolicyRequire
 public class ResetPasswordPolicyRequirementFactoryTests
 {
     [Theory, BitAutoData]
-    public void AutoEnrollAdministration_WithNoPolicies_ReturnsFalse(SutProvider<ResetPasswordPolicyRequirementFactory> sutProvider)
+    public void AutoEnroll_WithNoPolicies_IsEmpty(SutProvider<ResetPasswordPolicyRequirementFactory> sutProvider)
     {
         var actual = sutProvider.Sut.Create([]);
 
-        Assert.False(actual.AutoEnrollEnabled);
+        Assert.True(actual.AutoEnroll.IsNullOrEmpty());
     }
 
     [Theory, BitAutoData]
-    public void AutoEnrollAdministration_WithResetPasswordPolices_ReturnsTrue(
+    public void AutoEnrollAdministration_WithAnyResetPasswordPolices_ReturnsEnabledOrganizationIds(
```

Comment:
I recommend asserting against the public interface - the `AutoEnrollEnabled` method - rather than the internal list.